postsView.test.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fs = require('fs');
+const PostsView = require('./postsView');
+const Posts = require('./postsModel');
+const ChitterAPI = require('./chitterAPI');
+require('jest-fetch-mock').enableMocks();
+let api;
+let posts;
+let view;
```

Comment:
Is there a reason to why you declare them here and then give them values in the before each?